### ISSUE TITLE
relation between Art Theme and its content

### ISSUE THREAD

**Issue by thea-m** [BetaMasaheft/Documentation#1139]
just to clarify: is saws:contains the right relation between an Art Theme record and the persons depicted in it? As in AT1003Arrest: 
`<relation name="saws:contains"  active="AT1003Arrest" passive="PRS5684JesusCh">`
I had used ecrm:P129_is_about for the same purpose (having mentally reserved saws:contains for work records), so maybe others might also be confused and it would be good to put it in the guidelines.
@PietroLiuzzo ?

**Comment by eu-genia** [comment]
I agree it should be preferably ecrm:P129_is_about as we use for Work records written about someone, e.g.
```
<relation name="ecrm:P129_is_about"
                         active="LIT1113Anqasa"
                         passive="PRS6819Mary"/>
```


"saws:contains" should be limited to the cases when really parts of a whole are included (could potentially be not only works I guess)
